ecs-cli/modules/cli/compose/service/service.go

```diff
@@ -11,17 +11,20 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package ecs
+package service
```

Comment:
should we move service and task under entity? Because they are implementations of entity